packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx

```diff
@@ -0,0 +1,146 @@
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={headerCellStyle}>{t('status')}</td>
+                    <td css={contentCellStyle}>
+                        <WFOProcessStatusBadge
+                            processStatus={subscriptionDetailProcess.lastStatus}
+                        />
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={headerCellStyle}>{t('startedAt')}</td>
+                    <td css={contentCellStyle}>
+                        {parseDateToLocaleDateTimeString(
+                            parseDate(subscriptionDetailProcess.startedAt),
+                        )}
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={lastHeaderCellStyle}>{t('startedBy')}</td>
+                    <td css={lastContentCellStyle}>
+                        {subscriptionDetailProcess.createdBy}
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+            </table>
+        </div>
+    );
+};
+
+interface WfoRenderSubscriptionProcess {
+    subscriptionDetailProcess: SubscriptionDetailProcess;
+}
+const WfoRenderSubscriptionProcess = ({
+    subscriptionDetailProcess,
+}: WfoRenderSubscriptionProcess) => {
+    const { timeLineStyle, workflowTargetStyle } =
+        useWithOrchestratorTheme(getStyles);
+
+    return (
+        <EuiComment
+            username={subscriptionDetailProcess.workflowTarget ?? ''}
+            timelineAvatarAriaLabel={subscriptionDetailProcess.workflowName}
+            timelineAvatar={
+                <EuiAvatar name={subscriptionDetailProcess.workflowTarget} />
+            }
+        >
+            <div css={timeLineStyle}>
+                <EuiText css={workflowTargetStyle}>
+                    {upperCaseFirstChar(
+                        subscriptionDetailProcess.workflowTarget,
+                    )}
+                </EuiText>
+                <EuiText>{subscriptionDetailProcess.workflowName}</EuiText>
+            </div>
+
+            <WfoProcessCard
+                subscriptionDetailProcess={subscriptionDetailProcess}
+            />
+        </EuiComment>
+    );
+};
+
+interface WfoProcessesTimelineProps {
+    subscriptionDetailProcesses: SubscriptionDetailProcess[];
+}
+
+export const WfoProcessesTimeline = ({
+    subscriptionDetailProcesses,
+}: WfoProcessesTimelineProps) => {
+    return (
+        <>
+            <EuiSpacer size={'m'} />
+            {!subscriptionDetailProcesses && <WFOLoading />}
+            <EuiCommentList aria-label="Processes">
+                {subscriptionDetailProcesses && (
+                    <EuiCommentList aria-label="Processes">
```

Comment:
Old style naming.

Probably better to stick to new for now; and do the rename in one single ticket.
(it will make the transformation easier)

Reply:
See previous